### DISCUSSION TITLE
Broken links in Provision/DataPipeline pipelines

### DISCUSSION THREAD

**Discussion by vj4a** [Sunbird-Ed/Community#653] [Installation] [answered]
Hello @santhosh-tg , @gandham-santhosh ,

We faced these errors due to broken links. Kindly provide the currently active one in use for the below task.

**1.Provision/DataPipeline/InfluxDB:**
 
```
TASK [influxdb : Download InfluxDB-Relay .deb package from S3] *****************
fatal: [10.0.6.14]: FAILED! => {
    "changed": false,
    "dest": "/tmp/influxdb-relay.deb",
    "elapsed": 0,
    "response": "HTTP Error 404: Not Found",
    "status_code": 404,
    "url": "https://s3.ap-south-1.amazonaws.com/ekstep-third-party-libraries/influxdb-relay/influxdb-relay_535181e_amd64.deb"
}

MSG: Request failed
```

 Provided s3 [URL ](https://s3.ap-south-1.amazonaws.com/ekstep-third-party-libraries/influxdb-relay/influxdb-relay_535181e_amd64.deb) is broken. Kindly share the current URL for these libraries.

**2.Provision/DataPipeline/AnalyticsSpark:**

```
TASK [analytics-spark-provision : Download Kafka-2.11] *************************
fatal: [10.0.6.14]: FAILED! => {
    "changed": false,
    "dest": "/mount/data/analytics/kafka_2.11-0.10.1.0.tgz",
    "elapsed": 0,
    "url": "http://downloads.mesosphere.com/kafka/assets/kafka_2.11-0.10.1.0.tgz"
}

MSG: Request failed
```

The mesospehere link is not active now. We will use the archive [link ](https://archive.apache.org/dist/kafka/0.10.1.0/kafka_2.11-0.10.1.0.tgz) from apache to download the same package. Please let know of any change in that if needed.

**Comment by santhosh-tg** [comment, marked as answer]
Hi @vj4a  

1.Provision/DataPipeline/InfluxDB: - We don't need this job anymore. We have updated the documentation to remove it.
2. Provision/dev/DataPipeline/AnalyticsSpark - this is being still tested for OCI. You can use this forked repo and branch until its pushed to main repo https://github.com/sowmya-dixit/sunbird-data-pipeline  , branch:  **release-5.2.0**

**Comment by mohitga** [comment]
@vj4 is this issue resolved?

**Reply by vj4a** [reply]
@mohitga , This is resolved. Marking as closed.
